### DISCUSSION TITLE
npmjs.org page crash in chrome v100

### DISCUSSION THREAD

**Discussion by akinazuki** [npm/feedback#663] [Website] [answered]
When i click this link in Google Chrome,the Browser will crash suddenly
but other page will not crash

https://www.npmjs.com/search?q=async

My Google Chrome Version is 100.0.4896.60 (arm64) 
Hardware Info: 
MacBook Pro 14inch (Apple M1 Pro)
macOS version: 12.3.1
UserAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36

https://user-images.githubusercontent.com/43605695/161930214-1ece4786-3767-4651-8d6f-216070eac176.mp4

even crash in privacy mode

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/43605695/161930190-79dab930-5fd7-4808-a2ef-de890c677690.png">

**Comment by monishcm** [comment]
Thanks for the feedback. We will have a look.

**Comment by MylesBorins** [comment, marked as answer]
This should be fixed now. Looks like it was a chrome bug, we are trying to figure out a minimal reproduction to report to the chrome team
